From 03f831b6c890c63c4b223c350e8765de785f7786 Mon Sep 17 00:00:00 2001
From: Alex Zhao <zzc@rock-chips.com>
Date: Tue, 31 Oct 2017 17:33:30 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: wifi work as ko module

select CFG80211 and MAC80211 when CONFIG_WIFI_BUILD_MODULE is set

Change-Id: I42acf5d554223b13c0c689b8edae15b4a7a57a70
Signed-off-by: Alex Zhao <zzc@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/rockchip_wlan/Kconfig b/drivers/net/wireless/rockchip_wlan/Kconfig
index 41f120ef8356..a74ad8efbfd1 100644
--- a/drivers/net/wireless/rockchip_wlan/Kconfig
+++ b/drivers/net/wireless/rockchip_wlan/Kconfig
@@ -7,6 +7,8 @@ if WL_ROCKCHIP
 
 config WIFI_BUILD_MODULE
 	bool "build wifi ko modules"
+	select CFG80211
+	select MAC80211
 	default n
 	---help---
 	  Wifi drivers will compile as ko module
-- 
2.35.3

